### ISSUE TITLE
No ay of searching higher level taxonomic classification for fishes

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4600]
**No ay of searching higher level taxonomic classification for fishes**

I recently ran this search to try and find all fish records from Nepal - https://www.gbif.org/occurrence/search?basis_of_record=PRESERVED_SPECIMEN&country=NP&occurrence_status=present.  When drilling down into scientific name under Chjordata I realized that there is no entry for fishes and I am wondering why?  This obviously has something to do with the taxonomic authority you are using or a mismatch of some kind.  Can you outline why this is the case and how we can fix this?

Thanks

Andy

-----

Github user: @acbentley
User: [See in registry](https://www.gbif.org/api/feedback/user/2370fe33a2e023a9a8bbe786a15e06bc:13a75a1a3cbf7a5c7d31579947f0b8b4b6828c86eb774041c36e96f8bd0142031f345bf0cddfcf3349e31e12d0a6827cec20d9b7c0510eaf76b591f024f412de9503ef20eac8cc281fc29b64799c85aa9dc660c378ae51fa4ee8b8fbbdfe0cb498d1ea2e2afa3e7826100b08de382d2d4623c953e293d8c32f83322369029a7a) - [Send email](https://www.gbif.org/api/feedback/user/mailto/2370fe33a2e023a9a8bbe786a15e06bc:13a75a1a3cbf7a5c7d31579947f0b8b4b6828c86eb774041c36e96f8bd0142031f345bf0cddfcf3349e31e12d0a6827cec20d9b7c0510eaf76b591f024f412de9503ef20eac8cc281fc29b64799c85aa9dc660c378ae51fa4ee8b8fbbdfe0cb498d1ea2e2afa3e7826100b08de382d2d4623c953e293d8c32f83322369029a7a)
System: Chrome 110.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/search?basis_of_record=PRESERVED_SPECIMEN&country=NP&occurrence_status=present
Window size: width 1600 - height 1057
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-03-08T18:58:56.100Z',to:'2023-03-08T19:04:56.100Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-03-08T18:58:56.100Z',to:'2023-03-08T19:04:56.100Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
@acbentley, you are right, because there no longer an Actinopterygii class in the backbone taxonomy (see https://github.com/gbif/checklistbank/issues/79 and https://github.com/gbif/pipelines/issues/850), users need to look for the fish orders.

It looks like there are 2,533 preserved specimen occurrences of fish in Nepal:
https://www.gbif.org/occurrence/search?basis_of_record=PRESERVED_SPECIMEN&country=NP&taxon_key=1103&taxon_key=1104&taxon_key=494&taxon_key=495&taxon_key=1105&taxon_key=496&taxon_key=497&taxon_key=1106&taxon_key=498&taxon_key=499&taxon_key=1107&taxon_key=537&taxon_key=538&taxon_key=1153&taxon_key=547&taxon_key=1162&taxon_key=548&taxon_key=549&taxon_key=550&taxon_key=1163&taxon_key=1164&taxon_key=1165&taxon_key=1166&taxon_key=1167&taxon_key=1305&taxon_key=1067&taxon_key=1306&taxon_key=1307&taxon_key=1308&taxon_key=1068&taxon_key=1069&taxon_key=587&taxon_key=1310&taxon_key=588&taxon_key=589&taxon_key=1311&taxon_key=1312&taxon_key=1313&taxon_key=590&taxon_key=708&taxon_key=890&taxon_key=774&taxon_key=889&taxon_key=773&taxon_key=772&taxon_key=888&occurrence_status=present
